plugins/pulp_rpm/plugins/importers/yum/config_validate.py

```diff
@@ -25,3 +46,69 @@ def validate(config):
             msg += failure_message + '\n'
         msg = msg.rstrip()  # remove the trailing \n
         return False, msg
+
+
+def _validate_allow_keys(repo, config, error_messages):
```

Comment:
I think we should start by letting the user manage the list of keys, including the ability to remove them, and let the user decide if/when it is appropriate or useful to remove a key. They are free to be very strict about never removing keys.

@bmbouter you are thinking ahead about pulp taking responsibility for ongoing re-validation of content already in the repo, which I don't think we need to tackle at this time. To make an analogy, I think we should be a bouncer at the door to a repo. The bouncer has rules about who can get in. At any time, the owner can tell the bouncer to change the rules, or make exceptions for specific people. But the bouncer is not required to re-evaluate everyone who is already inside when the rules change, or when exceptions happen.